File: prime_numbers.py
from math import sqrt

number = int(input("Number : "))


if number == 1 :
    print("This number is one (not prime and not composite)")
    exit()

elif number == 0:
    print("This number is zero")
    exit()
    
else :

    for i in range(2 , int(sqrt(number)) + 1):
        operation = number % i

        if operation == 0 :
            print("this number is a composite number")
            exit()

print("this number is a prime number")

File: PrimeNumbersCPP/PrimeNumbers/PrimeNumbers.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int number;
    cout << "Number : ";
    cin >> number;

   

    if (number == 1) {
        cout << "This number is one (not prime and not composite)";
        return 0;

    }
    else {
        for (int i = 2; i <= (int)sqrt(number); i++) {

            int operation = number % i;

            if (i != number && operation == 0) {
                cout << "this number is a composite number";
                return 0;
            }
        }
    }



    cout << "this number is a prime number";



    return 0;

}

